### PR TITLE
Chatbox details v.02

### PR DESCRIPTION
**Pull request by wowgr8**
## Description

This code will add the following to GameChat.js:
- Emoji-picker-react
- Left aligned messages
- Gold text for message sender
- Chatbox height increase
- Chatbox width decrease
- Resolve input field repeat
- styling

GameRoom.js:
- Room number relocation + gold color
- player name gold color
- updated list from branch `game-room-player-list`

GameLobby.js:
- temporary background image

## Related Issue

Emoji component's width does not expand to full-width - WIP

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### After

image

image

### CODE REVIEW

File: src/components/GameLobby.js [hunk 1]
```diff
@@ -56,28 +46,28 @@ function GameLobby({ userName, gamesWon, _id }) {
...
-    //       players: ["name4", "name5", "name6"]
-    //    }]
     socket.on("available_rooms", (rooms) => {
       setAvailableRooms(rooms);
     });
   }, [socket]);
 
+  // TEMP background image - liable to change or be moved into App.js, across the entire app
+  const backgroundImage = {
+    backgroundImage: `url(${require('/Users/cesar/code-the-dream/practicum/bb-practicum-team-2-front/src/img/forest-bg.jpg')})`,
```
Comment (frankstepanski) [inline src/components/GameLobby.js]: Where is this location coming from? Where is **/Users** path? this needs to be either in the **public** folder or in the **src** folder.

**Review by frankstepanski — CHANGES_REQUESTED** [review]
Need to move the images to either the public or src folder of the app

File: src/components/GameLobby.js (hunk 1 above)
Comment (wowgr8) [reply src/components/GameLobby.js]: Whoops, I used the relative path by accident. I just updated it to the proper path.

File: src/components/GameRoom.js [hunk 2]
```diff
@@ -137,57 +134,65 @@ function GameRoom({ room, setInRoom, userName, host, gamesWon,_id }) {
...
+                }}
+              >
+                - {player}
+              </li>
+            </div>
+          ))}
+        </ul>
+        {console.log("players", players)}
+        <button onClick={leaveRoom}>Leave Room</button>
+        <button onClick={disconnectRoom}>Disconnect</button>
```
Comment (mautoranzo) [inline src/components/GameRoom.js]: The button disconnect is not necessary any more, now we have the Logout btn (line 165 this same file)

**Review by mautoranzo — CHANGES_REQUESTED** [review]
Nice job with  the emojis and the send icon!
Just a small correction in line 187 GameRoom component 
Also I suggest to change colors on the text is hard to read, but that can be another branch I think.

**Review by frankstepanski — APPROVED** [review]
Nice job

**Pull request merged**